lib/broccoli/glimmer-app.ts

```diff
@@ -253,6 +252,19 @@ export default class GlimmerApp extends AbstractBuild {
       srcTree = addonProcessTree(this.project, 'preprocessTree', 'src', srcTree);
     }
 
+    let stylesTree = options.trees && options.trees.styles;
+
+    if (typeof stylesTree === 'string') {
+      stylesTree = resolveLocal(root, stylesTree);
```

Comment:
I noticed that the `src` trees above are wrapped by a `new WatchedDir(dir)`, do I have to do this with `stylesTree` too?